### ISSUE TITLE
use the latest url in github releases

### ISSUE THREAD

**Issue by brianmorton** [brianmorton/cask-automation#12]

**Comment by core-code** [comment]
is this even possible? i discarded github appcasts as they didn't seem to provide a way to differentiate between 'latest release' an 'pre-release' releases

**Comment by brianmorton** [comment]
yes, strip the `.atom` off the appcast and replace it with `/latest`

**Comment by core-code** [comment]
nice, thanks! it yields the full website instead of something that could be more easily used in automation but its still much better than nothing

**Comment by core-code** [comment]
apparently the API can be used too:
https://api.github.com/repos/homebrew/homebrew-cask/releases/latest

**Comment by brianbrownton** [comment]
@core-code excellent find!!
